runironic-inspector.sh

```diff
@@ -11,6 +12,7 @@ export IRONIC_INSECURE=${IRONIC_INSECURE:-false}
 export IRONIC_INSPECTOR_CACERT_FILE=/certs/ca/ironic-inspector/tls.crt
 export IRONIC_INSPECTOR_CERT_FILE=/certs/ironic-inspector/tls.crt
 export IRONIC_INSPECTOR_KEY_FILE=/certs/ironic-inspector/tls.key
+export INSPECTOR_REVERSE_PROXY_SETUP=${INSPECTOR_REVERSE_PROXY_SETUP:-"true"}
```

Comment:
Is there a reason to keep this optional?

Reply:
Yes, since adding reverse proxy results in adding more delay to the network, so I suppose we should allow user to choose whether use it or not.